dss/config.py

```diff
@@ -348,6 +350,35 @@ def debug_level() -> int:
         """
         return int(os.environ.get('DSS_DEBUG', '0'))
 
+    @staticmethod
+    def get_openid_provider():
+        return Config._get_required_envvar("OPEN_ID_PROVIDER")
+
+    @staticmethod
+    def get_trusted_google_projects():
+        if Config._TRUSTED_GOOGLE_PROJECTS is None:
+            Config._TRUSTED_GOOGLE_PROJECTS = [x for x in Config.get_allowed_email_domains().split()
+                                               if x.endswith("iam.gserviceaccount.com")]
+        return Config._TRUSTED_GOOGLE_PROJECTS
+
+    @staticmethod
+    def get_audience():
+        if Config._JWT_AUDIENCE is None:
+            audience = Config._get_required_envvar("AUDIENCE")
+            Config._JWT_AUDIENCE = audience.split(',')
+        return Config._JWT_AUDIENCE
+
+    @staticmethod
+    def get_group_claim():
```

Comment:
Same here. Please rename group_claim to jwt_group_claim_name or something else more descriptive in all identifiers.

Reply:
changed to OIDC_GROUP_CLAIM